src/app/(pages)/fhirServers/page.tsx

```diff
@@ -364,6 +368,32 @@ const FhirServers: React.FC = () => {
   const handleSave = async (authData: AuthData) => {
     const connectionResult = await testFhirConnection(serverUrl);
 
+    if (defaultServer) {
+      await Promise.all(
+        fhirServers
```

Comment:
Is this part just turning default off of the previously default server before setting it on the currently selected one? IIRC, I set up a database constraint such that only one row can have default as true, so I thought setting it on one row will automatically flip it off the other. I see now in testing that that doesn't actually happen though, so I think you had the right idea adding this!!